user.py

```diff
@@ -82,13 +81,28 @@ def check_username(username, db_username):
     def get_user_by_id(cls, userid):
         user = cls.Users.get(userid, None)
         return user
+    
+    @classmethod
+    def get_userid(cls, displayname):
+        for _, user in cls.Users.items():
+            # print(user)
+            if user.displayName == displayname:
+                # print(f'e{user}')
+                userid = user.uuid
+        return userid
+    
+    @classmethod
+    def get_display(cls, uuid):
+        user = cls.Users.get(uuid, None)
+        displayname = user.displayName
+        return displayname
 
     @classmethod
     def add_user_class(cls, username, user, userid):
         user_class = cls(username, user, userid)
         database.set_online(userid, False)
         cls.Users.update({userid: user_class})
-        tupple = (userid, user['displayName'], user['SPermission'][0])
+        (userid, user['displayName'], user['SPermission'][0])
```

Comment:
what is this line doing here if we aren't assigning it to a value anymore?

Reply:
one not so rare case where I reworked something and was blind